src/templates/ja/2019/base.html

```diff
@@ -30,103 +30,103 @@
 {% block part %}部{% endblock %}
 {% block chapter %}章{% endblock %}
 
-{% block start_exploring %}Start exploring{% endblock %}
+{% block start_exploring %}探索を開始{% endblock %}
 
-{% block menu_title %}Page menu{% endblock %}
-{% block open_the_menu %}Open the menu{% endblock %}
-{% block close_the_menu %}Close the menu{% endblock %}
+{% block menu_title %}ページメニュー{% endblock %}
+{% block open_the_menu %}メニューを開く{% endblock %}
+{% block close_the_menu %}メニューを閉じる{% endblock %}
 
-{% block index_nav_title %}Chapter table of contents{% endblock %}
-{% block index_title %}Index{% endblock %}
-{% block open_the_index %}Open the Table of Contents{% endblock %}
-{% block close_the_index %}Close the Table of Contents{% endblock %}
+{% block index_nav_title %}章の目次{% endblock %}
+{% block index_title %}インデックス{% endblock %}
+{% block open_the_index %}目次を開く{% endblock %}
+{% block close_the_index %}目次を閉じる{% endblock %}
 
-{% block open %}Open{% endblock %}
-{% block close %}Close{% endblock %}
+{% block open %}開く{% endblock %}
+{% block close %}閉じる{% endblock %}
 
 {% macro onTwitter(twitterHandle) %}@{{twitterHandle}} on Twitter{% endmacro %}
 {% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} on GitHub{% endmacro %}
```

Comment:
Missed a translation here?

Google Translate is telling me it might be this:
```suggestion
{% macro onTwitter(twitterHandle) %}@{{twitterHandle}}のTwitter{% endmacro %}
{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}}のGitHub{% endmacro %}
```

Reply:
I think this part don't need translate.

Reply:
Really? OK if you don't think so.

[Here's](https://almanac.httparchive.org/ja/2019/contributors#ksakae) where it appears btw:

![image](https://user-images.githubusercontent.com/10931297/80201206-afce6700-861b-11ea-9719-d26be5553842.png)